doc/data/messages/r/repeated-keyword/bad.py

```diff
@@ -0,0 +1,2 @@
+print("Hello ", "orld", sep="W", end=" !", **{"end": " :("})  # [repeated-keyword]
+print("Hello ", "orld", **{"sep": "W", "end": " !"}, **{"end": " :("})  # [repeated-keyword]
```

Comment:
Interesting. These two lines don't raise an error.

Reply:
```python
def print(*args, **kwargs):
    pass

print("Hello ", "orld", sep="W", end=" !", **{"end": " :("})  # [repeated-keyword]
print("Hello ", "orld", **{"sep": "W", "end": " !"}, **{"end": " :("}) # [repeated-keyword]
```

worked.

Reply:
```python
def f(*args, **kwargs):
    pass

f("Hello ", "orld", sep="W", end=" !", **{"end": " :("})  # [repeated-keyword]
f("Hello ", "orld", **{"sep": "W", "end": " !"}, **{"end": " :("}) # [repeated-keyword]
```

also worked.

Reply:
Hurgh, look like a bug in pylint. Maybe we can't determine the signature of print properly.

Reply:
I opened #6856, but maybe there's another well known function that exists without the bug ?

Reply:
Thanks! How about `open`?

Reply:
Well this is not raising anything either:
```python
with open("file", "r", encoding="utf8", **{"encoding": "ascii"}) as f:
    f.read()
```
Let's keep your original example.